frontend/src/pages/api/scrape.ts

```diff
@@ -0,0 +1,90 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
@ashin-czi Let's remove this for now, since we aren't using Wikipedia descriptions anymore.

Remove the wikipedia hooks from `cellCards.ts` as well. <3 thank you!